modules/custom/az_event/az_event_trellis/az_event_trellis.install

```diff
@@ -11,3 +11,11 @@
 function az_event_trellis_update_1020801() {
   \Drupal::service('module_installer')->install(['views_remote_data']);
 }
+
+/**
+ * Ensure az_recurring_import_rule entity type is installed.
+ */
+function az_event_trellis_update_1020802() {
```

Comment:
Just sanity checking the update number here.  If this is a 2.10.x only change this number should probably be updated:
```suggestion
function az_event_trellis_update_1021001() {
```